app/src/stores/insights.ts

```diff
@@ -430,12 +430,12 @@ export const useInsightsStore = defineStore('insightsStore', () => {
 				);
 			}
 
-			if (edits.update) {
+			if (edits.update?.length > 0) {
 				requests.push(api.patch(`/panels`, edits.update));
 			}
 
-			if (edits.delete) {
-				requests.push(api.delete(`/panels`, { data: edits.delete }));
+			if (edits.delete?.length > 0) {
+				requests.push(api.patch(`/panels`, { data: { _deleted: true }, keys: edits.delete }));
```

Comment:
The other way to fix this would be to call the [graphql version of delete](https://docs.directus.io/reference/system/panels.html#graphql-7)
It just seemed a bit off to use the `panels` endpoint everywhere else and then `/graphql/system` here.  Also, I like the batch update. :-)

I used the Couchdb as a reference this as they allow a [delete in a bulk update](https://docs.couchdb.org/en/latest/api/database/bulk-api.html#post--db-_bulk_docs).